### PR TITLE
Feat/create blog model

### PR DESCRIPTION
**Pull request by Monterazo**
## What I did

- Create Blogpost.js file inside models folder at server
- Insert the blog model at blogpost.js
- Insert the source /api/blogpost at index.js
- Fixed the file Post.js

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `cd server/`
   1. `node index.js`
  

1. In your browser
   1. Open localhost [http://localhost:3001/admin/blog-posts](url)
   1.  Then press Blog Post
   1. Check if image, date, description and link fields are available
   1. Then, save the model
   1. Go back to [http://localhost:3001/admin/blog-posts](url)
   1. Check if the blog archive has the correct title (id)
   1. Go to [http://localhost:3001/api/blogpost](url)
   1. Check if error 500 is not showing up
   1. Check if the code is showing up

&nbsp;
*

### CODE REVIEW

File: server/models/Post.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ Post.add({
     index: true,
   },
   image: {
-    type: Types.CloudinaryImages,
+    type: Types.CloudinaryImage,
```
Comment (raissaheimann) [inline server/models/Post.js:15]: Put the "s" because this will probably break the front end

File: server/models/Blog.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  name: {
+    type: Types.Text,
+    required: true,
+    initial: true,
+    index: true,
+  },
+  image: {
+    type: Types.CloudinaryImage,
+    require: true,
+
```
Comment (viniciusscala) [inline server/models/Blog.js]: Erase this blank line

File: server/models/Blog.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  },
+  date: {
+    type: Types.Date,
+  },
+
+  description: {
```
Comment (viniciusscala) [inline server/models/Blog.js:25]: I think it's better to change this name to content, since this is the beggining of the blog post

**Review by viniciusscala — CHANGES_REQUESTED** [review]
Try to make everything required

**Review by guilhermefxs — COMMENTED** [review]
Everything looks good

**Review by guilhermefxs — APPROVED** [review]

**Review by raissaheimann — APPROVED** [review]

**Review by viniciusscala — APPROVED** [review]

**Pull request merged**